### ISSUE TITLE
Wider dependency requirements

### ISSUE THREAD

**Issue by Rishit-dagli** [Rishit-dagli/Gradient-Centralization-TensorFlow#14]
The package as of now to be installed requires  `tensorflow ~= 2.4.0` and `keras ~= 2.4.0`. It turns out that this is sometimes problematic for folks who have custom installations of TensorFlow and a winder requirement could be set up.

**Comment by Rishit-dagli** [comment]
Fixed via commits 770de711290d230f96feed5a2e5d073ebedfa23a and 4ee65d670dbcf8b0475c8bf417edaedc3663fec8 .
